src/cli.spec.ts

```diff
@@ -0,0 +1,387 @@
+import { removeFromGlobalConfig } from './utils/config'
+import { checkAndSetProjectDirectory, cli } from './cli'
+import {
+  createTestApp,
+  createTestGlobalTarget,
+  removeTestApp
+} from './utils/test'
+import { generateTimestamp } from './utils/utils'
+import * as mainModule from './main'
+import { ReturnCode } from './main'
+import { Command } from './utils/command'
+import { Target } from '@sasjs/utils'
+
+describe('checkAndSetProjectDirectory', () => {
+  let target: Target
+
+  beforeEach(async (done) => {
+    const appName = `cli-tests-${generateTimestamp()}`
+    await createTestApp(__dirname, appName)
+    target = await createTestGlobalTarget(
+      appName,
+      `/Public/app/cli-tests/${appName}`
+    )
+    done()
+  })
+
+  afterEach(async (done) => {
+    await removeTestApp(__dirname, target.name)
+    await removeFromGlobalConfig(target.name)
+    jest.clearAllMocks()
```

Comment:
Do we still need to clear it here (we've set `clearMocks: true` in jest config)?

Reply:
Keeping it there just in case.